PagesGallery/PagesGallery/PagesGallery.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <ProduceReferenceAssembly>True</ProduceReferenceAssembly>
```

Comment:
move this to Directory.build.props ?

Reply:
You probably only want this for netstandard projects, let me see if there is a condition or something I could check.